basxconnect/core/wizards/add_person.py

```diff
@@ -340,6 +372,12 @@ def done(self, form_list, **kwargs):
                 if k in ("address", "city", "postcode", "country")
             }
         )
+        if "email" in list(form_list)[-1].cleaned_data:
```

Comment:
It would help me as a reader if you could extract a variable for `list(form_list)[-1]` and give it an understandable name.
Would it be possible to use `get_cleaned_data_for_step("Confirmation")` instead of `list(form_list)[-1].cleaned_data`?

Reply:
I do not remember the exact reason but I think using ```form_list``` is the better option. The form handling with the wizard tools is a bit fragile, so I just refactored but did not change the behaviour.